utilities/pipelines/sharedScripts/Get-ScopeOfTemplateFile.ps1

```diff
@@ -0,0 +1,53 @@
+﻿<#
+.SYNOPSIS
+Get the scope of the given template file
+
+.DESCRIPTION
+Get the scope of the given template file (supports ARM & Bicep)
+Will return either
+- resourceGroup
+- subscription
+- managementGroup
+- tenant
+
+.PARAMETER TemplateFilePath
+Mandatory. The path of the template file
+
+.EXAMPLE
+Get-ScopeOfTemplateFile -TemplateFilePath 'C:/deploy.json'
+
+Get the scope of the given deploy.json template.
+#>
+function Get-ScopeOfTemplateFile {
+
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [Alias('Path')]
+        [string] $TemplateFilePath
+    )
+
+    if ((Split-Path $templateFilePath -Extension) -eq '.bicep') {
+        # Bicep
+        $bicepContent = Get-Content $templateFilePath
+        $bicepScope = $bicepContent | Where-Object { $_ -like '*targetscope =*' }
+        if (-not $bicepScope) {
+            $deploymentScope = 'resourceGroup'
+        } else {
+            $deploymentScope = $bicepScope.ToLower().Split('=')[-1].Replace("'", '').Trim()
```

Comment:
Am I right that this will cause the function to return lowercase instead of camelCase as documented above? Should we align also ARM to the following instead? 
- resourcegroup
- subscription
- managementgroup
- tenant

Reply:
done. I'm sure there was a good reason